###############################################################################
SHORT DESCRIPTION:
###############################################################################
QEMU remove unused networking support

###############################################################################
LONG DESCRIPTION:
###############################################################################
In an effort to minimize the size of QEMU, we remove unused features.  Here, we
build-time disable unused network support.  These cannot be disabled in the
configure script, so we set them to 'n' in Makefile.objs.

###############################################################################
PATCHES:
###############################################################################

--- a/net/Makefile.objs
+++ b/net/Makefile.objs
@@ -1,6 +1,6 @@
 common-obj-y = net.o queue.o checksum.o util.o hub.o
 common-obj-y += socket.o
-common-obj-y += dump.o
+common-obj-n += dump.o
 common-obj-y += eth.o
 common-obj-$(CONFIG_L2TPV3) += l2tpv3.o
 common-obj-$(CONFIG_POSIX) += vhost-user.o
@@ -8,8 +8,8 @@ common-obj-$(CONFIG_SLIRP) += slirp.o
 common-obj-$(CONFIG_VDE) += vde.o
 common-obj-$(CONFIG_NETMAP) += netmap.o
 common-obj-y += filter.o
-common-obj-y += filter-buffer.o
-common-obj-y += filter-mirror.o
+common-obj-n += filter-buffer.o
+common-obj-n += filter-mirror.o
 common-obj-y += colo-compare.o
 common-obj-y += colo.o
 common-obj-y += filter-rewriter.o
